utils/oscap-bootc

```diff
@@ -0,0 +1,122 @@
+        "image.")
+    parser.add_argument(
+        "--profile",
+        help="The name of profile to be evaluated")
+    parser.add_argument(
+        "--tailoring-file",
+        help="Use given XCCDF Tailoring file")
+    parser.add_argument(
+        "--tailoring-id", metavar="COMPONENT_ID",
+        help="Use given DS component as XCCDF Tailoring file")
+    parser.add_argument(
+        "--results-arf",
+        help="Write ARF (result data stream) into file")
+    parser.add_argument(
+        "--report",
+        help="Write HTML report into file")
+    parser.add_argument(
+        "data_stream", metavar="DATA_STREAM",
+        help="Path to a SCAP source data stream, eg. "
+        "/usr/share/xml/scap/ssg/content/ssg-rhel10-ds.xml")
+    # Unfortunately, we can't add "--rule", "--skip-rule", or "--reference"
+    # because the "oscap xccdf generate fix" submodule doesn't support these
+    # options.
+    return parser.parse_args()
+
+
+def ensure_sce_installed():
+    query_cmd = ["rpm", "-q", "openscap-engine-sce"]
+    query_process = subprocess.run(query_cmd, capture_output=True)
+    if query_process.returncode != 0:
+        install_cmd = ["dnf", "-y", "install", "openscap-engine-sce"]
+        try:
+            subprocess.run(install_cmd, check=True, capture_output=True)
+        except subprocess.CalledProcessError as e:
+            print(e.stderr.decode("utf-8"), file=sys.stderr)
+
+
+def add_args(option_args_list, cmd):
+    for o, a in option_args_list:
+        if a:
+            cmd.append(o)
+            cmd.append(a)
+
+
+def add_common_args(args, cmd):
+    oal = [
+        ("--profile", args.profile),
+        ("--tailoring-file", args.tailoring_file),
+        ("--tailoring-id", args.tailoring_id)
+    ]
+    add_args(oal, cmd)
+
+
+def add_eval_args(args, cmd):
+    oal = [
+        ("--results-arf", args.results_arf),
+        ("--report", args.report),
+    ]
+    add_args(oal, cmd)
+
+
+def pre_scan_fix(args):
+    with tempfile.NamedTemporaryFile(delete=False) as remediation_script:
+        gen_fix_cmd = [
+            "oscap", "xccdf", "generate", "fix", "--fix-type", "bootc",
+            "--output", remediation_script.name]
+        add_common_args(args, gen_fix_cmd)
+        gen_fix_cmd.append(args.data_stream)
+        subprocess.run(gen_fix_cmd, check=True)
+        subprocess.run(["bash", remediation_script.name], check=True)
+
+
+def scan_and_remediate(args):
+    oscap_cmd = ["oscap", "xccdf", "eval", "--progress", "--remediate"]
+    add_common_args(args, oscap_cmd)
+    add_eval_args(args, oscap_cmd)
+    oscap_cmd.append(args.data_stream)
+    env = {"OSCAP_PREFERRED_ENGINE": "SCE", "OSCAP_BOOTC_BUILD": "YES"}
+    try:
+        subprocess.run(oscap_cmd, env=env, check=True)
+    except subprocess.CalledProcessError as e:
+        if e.returncode not in [0, 2]:
+            print(e, file=sys.stderr)
+
+
+def main():
+    args = parse_args()
+    ensure_sce_installed()
```

Comment:
It might be better if we do not install anything here and instead mention that `openscap-engine-sce` should be installed before using this script? Same as we mention `openscap-utils` and `scap-security-guide`. Another option is to make `openscap-engine-sce` a dependency of  `openscap-utils` package as without it the script won't work.